docs/docs-ref-autogen/outlook/office.appointmentread.yml

```diff
@@ -1111,6 +1171,19 @@ items:
 
       <tr><td>[Applicable Outlook
       mode](https://docs.microsoft.com/outlook/add-ins/#extension-points)</td><td>Appointment Attendee</td></tr></table>
+
+      #### Examples
+
+
+      ```javascript
+
+      if (Office.context.mailbox.item.itemType == Office.MailboxEnums.ItemType.Appointment) {
```

Comment:
This one might not be ok with the rough ==, I think I would change it to === or check with an engineer?